docs/ota.md

```diff
@@ -8,26 +8,946 @@ Ruters repositories.
+|------|----------------------------------|------|---------------------------------------------------------------------------|----------------------|----------|-------------------------|
+| DC1  | Door Control 1                   | FE4E | 3411 Status 2 of doors <br> 1820 Ramp/Wheel chairlift <br> 1821 Position of doors   | Doors open / closed  | 0000FE4E | telemetry/0000fe4e/json |
+| DC2  | Door Control 2                   | FDA5 | XXXX Lock Status Door N <br> XXXX Enable Status Door N <br> XXXX Open Status Door N | Alternative to DC1?  | 0000FDA5 | telemetry/0000fda5/json |
+| VDHR | High Resolution Vehicle Distance | FEC1 | 917 High resolution total vehicle distance                                | Supplements position | 0000FEC1 | telemetry/0000fec1/json |
+
+#### ITxPT
+These PGNs must be subscribed to when the buses are started (unless the subscription is persistent) through the AddPGN call:
+```xml
+<PGNReq>FE4E</PGNReq>
+<PGNReq>FDA5</PGNReq>
+<PGNReq>FEC1</PGNReq>
+```
+A service must be implemented by the operator's vendor that listens the UDP messages and converts them to MQTT messages.
+
+#### XML format (source)
+The message for all subscribed PGNs is sent to UDP as XML.
+```xml
+<FMStoIPDelivery>
+    <FMStoIP FMSVersion="03">
+        <frame Status="OK">
+            <PGN>FEF1</PGN>
+            <data>F35237C403501FFF</data>
+            <relativetime>5000</relativetime>
+            <SPN id="84">
+                <name>Wheel-Based Vehicle Speed</name>
+                <unit>km/h</unit>
+                <value>55.3</value>
+            </SPN>
+            <SPN id="597">
+                <name>Brake Switch</name>
+                <value>released</value>
+            </SPN>
+        </frame>
+        <frame Status="OK">
+            <PGN>F004</PGN>
+            <data>F35237C403501FFF</data>
+            <relativetime>5000</relativetime>
+            <SPN id="190">
+                <name>Engine Speed</name>
+                <unit>rpm</unit>
+                <value>854</value>
+            </SPN>
+        </frame>
+    </FMStoIP>
+</FMStoIPDelivery>
+```
+PGN FEF1 and F004 shown here are the examples found in the ITxPT spec.
+
+#### JSON format (MQTT)
+The message containing several PGNs is split up into several MQTT messages. SPNs are mapped as subids.
+
+**telemetry/0000fef1/json**
+```json
+{
+  "eventTimestamp": "2017-10-31T12:45:50Z",
+  "id": "0000FEF1",
+  "payloads": [
+    {
+      "subid": 84,
+      "name": "Wheel-Based Vehicle Speed",
+      "unit": "km/h",
+      "value": 55.3
+    },
+    {
+      "subid": 597,
+      "name": "Wheel-Based Vehicle Speed",
+      "value": "released"
+    }
+  ]
+}
+```
+
+**telemetry/0000f004/json**
+```json
+{
+  "eventTimestamp": "2017-10-31T12:45:50Z",
+  "id": "0000F004",
+  "payloads": [
+    {
+      "subid": 190,
+      "name": "Engine Speed",
+      "unit": "rpm",
+      "value": 854
+    }
+  ]
+}
+```
+
+#### Fields
+
+#### Telemetry
```

Comment:
is this supposed to be a sub-header to "fields"?

Reply:
good catch!